README.md

```diff
@@ -1,6 +1,10 @@
 # Azure SDK for Node.js
 
-[![NPM version](https://badge.fury.io/js/azure.png)](https://badge.fury.io/js/azure) [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-node.png?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-node)
+| Component | Status |
+| --------- | ------ |
+| Packages | [![NPM version](https://badge.fury.io/js/azure.png)](https://badge.fury.io/js/azure) |
```

Comment:
Feel free to omit this line. Even though this was the badge that already existed, we don't ship the azure "rollup" package anymore, so we don't need this badge.